### DISCUSSION TITLE
Add more maintainers to rust-bitcoincore-rpc

### DISCUSSION THREAD

**Discussion by sanket1729** [rust-bitcoin/rust-bitcoin#962] [General]
AFAIK, only @stevenroose maintains `rust-bitcoincore-rpc` now. The library is really handy for making sure that we are in consensus with bitcoin core. We use this in rust-miniscript to create complex miniscript transactions, spend them and check if they are accepted by the bitcoin core node. I suspect other projects also benefit strongly from this. For these reasons, many of these projects have added this to the CI checklist. 

Maintaining the crate is a fairly low amount of work to keep updating this every rust-bitcoin release(took me 10 mins for this PR) https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/217. There can be new feature requests, but they are really low priority compared to most things in the rust-bitcoin ecosystem. 

If we also integrate this with the rust-bitcoin cycle and add rust-bitcoin maintainers there, it will help downstream projects a lot. We don't need the code to be high quality in rust-bitcoincore-rpc, so maintenance is just chasing obvious compiler errors every rust-bitcoin release and making a prompt release.

@stevenroose what do you think about this?

**Comment by dpc** [comment]
I'd like to bump this. I just noticed that PRs are pilling up there. Issues pilling up is another story, but when PRs are pilling up that means there's a lot of motivated users, that are willing do the work and are blocked on maintainers, which is a wasted effort. Ideally PRs either get merged, or rejected and closed.

**Comment by tcharding** [comment]
I can help if needed.

**Comment by tcharding** [comment]
Steven is a bit quiet on GitHub thees days, I've emailed him to see if we an push this forwards.

**Comment by stevenroose** [comment]
Yeah the last half year has been a bit busy work- and life-wise. I have had some refactor work done some months ago and lost track of the repo since. I'm definitely open to adding an extra maintainer. How about I add Tobin?
